src/app/util/config.cc

```diff
@@ -173,12 +175,16 @@ bool SetLoggingOptions() {
       arg_levels.push_back(level.first);
   }
   // Set new global log-levels
-  log::SetGlobalLogLevels(arg_levels);
+  core::SetGlobalLogLevels(arg_levels);
   // Set other logging options
-  log::SetOptionLogToConsole(var_map["log-to-console"].as<bool>());
-  // TODO(anonimal): hack to ensure log-to-console is enabled in daemon mode
-  log::SetOptionLogToFile(var_map["log-to-file"].as<bool>());
-  log::SetOptionLogFileName(var_map["log-file-name"].as<std::string>());
+  core::SetOptionLogToConsole(VarMap["log-to-console"].as<bool>());
+  // TODO(unassigned): the following daemon test is a HACK to ensure that
+  // log-to-console is enabled in daemon mode (or else we'll boost.log segfault)
```

Comment:
Note: with `log-to-console` enabled, only the first handful of lines of output will be sent to console (in the parent process) before forking. File logging will remains intact throughout.